### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by yyunikov**
Sample app for using CALM for one-time enrollment:
- Reads card numbers from file (aliases and raw numbers are supported)
- Enrolls cards to CALM API one by one
- Possible to redact raw card numbers on test vault before enrollment

See README for more details.

### CODE REVIEW

File: routes/calm_outbound.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          phase: REQUEST
+          public_token_generator: UUID
+          targets:
+            - body
+          token_manager: PERSISTENT
+          transformer: JSON_PATH
+          transformer_config:
+            - $.account_number
+          transformer_config_map: null
+        - classifiers: {}
```
Comment (berdarm) [inline routes/calm_outbound.yaml:52]: Why do we need this filter?
image
I thought we're not receiving raw data if we aliased it previously

File: main.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    print(
+        f'Successfully finish one-time enrollment process. '
+        f'See "csv/updated_cards.csv" for enrolled card with their corresponding ID\'s.')
+
+
+if __name__ == "__main__":
+    vault_id = os.getenv('VAULT_ID')
+    vault_username = os.getenv('VAULT_USERNAME')
+    vault_password = os.getenv('VAULT_PASSWORD')
+    environment = os.getenv('ENVIRONMENT') if os.getenv('ENVIRONMENT') else 'sandbox'
```
Comment (berdarm) [inline main.py:115]:
```suggestion
    environment = os.getenv('ENVIRONMENT') or 'sandbox'
```

**Review by berdarm — APPROVED** [review]

File: routes/calm_outbound.yaml (hunk 1 above)
Comment (odovzhenko) [reply routes/calm_outbound.yaml:52]: I believe we always get raw data in response from CALM and alias in the customer vault.

Comment (yyunikov) [reply routes/calm_outbound.yaml:52]: hm... indeed. what does status code not equal 1234 means here?

File: main.py (hunk 2 above)
Comment (yyunikov) [reply main.py:115]: @odovzhenko Could you please apply suggestion in a follow up PR? thanks.

Comment (odovzhenko) [reply main.py:115]: Done.

File: routes/calm_outbound.yaml (hunk 1 above)
Comment (odovzhenko) [reply routes/calm_outbound.yaml:52]: Updated in the following PR.

**Pull request merged**